static/src/stylesheets/module/facia/_mixins.scss

```diff
@@ -22,15 +22,15 @@ $breakpoints: (
             margin-left: $gs-gutter;
             margin-right: $gs-gutter;
         }
-        @include mq(containerWidestMobile) {
```

Comment:
How are these changes relevant for this change? 

Given its a facia mixin I just want to confirm that it's changing something unexpected.